README.md

```diff
@@ -0,0 +1,21 @@
+# Certificate-Generator
+This application will automatically create &amp; mail the certificates using the Certificate image and the recipients details in an Excel sheet  
+
+## Getting Started
+### Prerequisites
+You need to have Python installed on your PC. If you do not have you can [install from here](https://www.python.org/downloads/)
+
+### Installation
+
+1. Clone the repo
+   ```sh
+   git clone https://github.com/githubcrce/Certificate-Generator.git
+   ```
+3. Install the Requirements
+   ```sh
+   pip install -r requirements.txt
+   ```
+4. Run the Certificate Generator
```

Comment:
Please mention the need of creating `credentials.json` and provide an outline for the same which will help other users before running the application